app.rb

```diff
@@ -0,0 +1,96 @@
+require './person'
+require './student'
+require './teacher'
+require './classroom'
+require './corrector'
+require './book'
+require './rental'
+
+class App
+  attr_reader :books, :people, :rentals
+
+  def initialize(books = [], people = [], rentals = [])
+    @books = books
+    @people = people
+    @rentals = rentals
+  end
+
+  def all_books
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def all_people
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
+  # rubocop:disable Metrics/MethodLength
```

Comment:
- [ ] Kindly remove this and use `rubocop.yml` to achieve this purpose in order to make your file clean. More so, commented-out code makes our work look messy. ℹ️